.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.13
+      - image: circleci/golang:1.15
```

Comment:
We should update this to build the current and most recent version of Go similar to the `otel` repo. I'll open a ticket for this.

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/issues/239

Reply:
you mean `golang:latest` ? Don't know, maybe it's better to have always a precise reference on which go version you are using

Reply:
No, I was referring to the fact that this should also be building for Go 1.14. I agree direct specifying of the version is preferred.